laravel-app/app/Http/Controllers/FollowController.php

```diff
@@ -3,11 +3,30 @@
 namespace App\Http\Controllers;
 
 use App\Models\User;
+use Illuminate\Contracts\View\View;
 use Illuminate\Http\RedirectResponse;
 
 class FollowController extends Controller
 {
-    public function follow(User $user): RedirectResponse
+    /**
+     * To display all the followers-following of the user
```

Comment:
please remove comments that it is already understandable, you over documenting your code

Reply:
done sir, thank you ❤️